src/app/feature/feedback/routeGuards.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from "@angular/core";
+import { CanActivate } from "@angular/router";
+
+@Injectable({ providedIn: "root" })
+export class RouterDisableGuard implements CanActivate {
```

Comment:
Adding router guards seemed to be the best way I could find to disable navigation. They can be applied/toggled to all routes, to either block or allow navigation to that route